packages/react/src/utilities/dom.ts

```diff
@@ -17,7 +17,7 @@ import { useDocument, useWindow, WindowProviderProps } from '@fluentui/react-win
  */
 export const useDocumentEx = () => {
   // eslint-disable-next-line no-restricted-globals
-  return useDocument() ?? typeof document !== 'undefined' ? document : undefined;
+  return useDocument() ?? (typeof document !== 'undefined' ? document : undefined);
```

Comment:
Would be a nice to add tests to cover these but also not a blocking issue.

Reply:
I'd love to contribute a test but I'm sadly not able to build the monorepo on my daily driver box (2019 MBP) to run jest

Reply:
No worries. Thanks for the contribution!